src/authentication.ts

```diff
@@ -161,7 +161,7 @@ export async function getUser(
   ctx: Context<Env>,
   securitySchemeName: SecuritySchemeName,
   bearer: string,
-  scopes: string[],
+  permissios: string[],
```

Comment:
typo - this is probably Portuguese :smile: 
```suggestion
  permissions: string[],
```

Reply:
ohh.. slopppy